### DISCUSSION TITLE
Disable autoscroll

### DISCUSSION THREAD

**Discussion by kevin9r** [dlsc-software-consulting-gmbh/KeyboardFX#9] [General]
Hello Dirk, 

is it possible to disable the automatically scroll of the screen? We have a form field which fits exactly in the middle of the screen. The keyboard is still scrolling the stack pane although there is enough space. Thanks for your information! 

And great work!

**Comment by dlemmermann** [comment]
Not sure if there is an API for it, yet, but sounds like it would make sense to have it. Can you provide a PR for it?
